contents/blog/sunsetting-helm-support-posthog.md

```diff
@@ -0,0 +1,43 @@
+---
+date: 2023-01-05
+title: "Sunsetting Kubernetes support for PostHog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["tim-glaser"]
+featuredImage: ../images/blog/green-blog-image.jpg
+featuredImageType: full
+categories: ["posthog-news"]
+---
+
+We're sunsetting support for our Kubernetes deployment for PostHog. Because we're an open source company, I want to be transparent about what this change means, and why we're making it. Around 3.5% of PostHog users currently use this type of deployment.
```

Comment:
```suggestion
We're sunsetting support for our Kubernetes deployment for PostHog. Because we're an open source company, I want to be transparent about what this change means, and why we're making it. Around 3.5% of PostHog users currently use this type of deployment. We will continue to support the docker deployment of PostHog.
```